qa/suites/deepsea/tier3/upgrade/tasks/upgrade.yaml

```diff
@@ -0,0 +1,27 @@
+tasks:
+        - deepsea.preflight:
+        - deepsea.orch:
+                stage: prep
+                alternative_defaults:
+                  - stage_prep_master: default-update-reboot
+                  - stage_prep_minion: default-update-reboot
+        - deepsea.orch:
+                stage: 1
+        - deepsea.policy:
+                profile: default
+        - deepsea.orch:
+                stage: 2
+        - deepsea.ceph_conf:
+        - deepsea.orch:
+                stage: 3
+        - exec:
+                client.salt_master:
+                        - 'ceph -s'
+        - deepsea.orch:
+                state_orch: ceph.maintenance.upgrade
```

Comment:
humor an old man and put "allow-reboot: true" here?

Reply:
in